examples/deployment-manager/autogen/singlevm/Kptfile

```diff
@@ -0,0 +1,18 @@
+apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
+metadata:
+  name: wordpress
```

Comment:
Can we replace `wordpress` (here and on L6) with some place holder like `YOUR_PACKAGE`?